src/main/java/com/fasterxml/jackson/databind/annotation/JsonPOJOBuilder.java

```diff
@@ -85,6 +85,6 @@ public Value(String buildMethodName, String withPrefix)
 	        this.withPrefix = withPrefix;
 	    }
 
-	    public static Value empty() { return EMPTY; }
```

Comment:
It looks like it was added in `2.9.0-pr1` so it is safe to change:
https://github.com/fasterxml/jackson-databind/commit/e45279370ee9d3977776159c6f23d67f05379613